modules/ding_staff/ding_staff.features.field_instance.inc

```diff
@@ -56,7 +56,9 @@ function ding_staff_field_default_field_instances() {
     'widget' => array(
       'active' => 1,
       'module' => 'options',
-      'settings' => array(),
+      'settings' => array(
+        'match_operator' => 'CONTAINS',
+      ),
```

Comment:
This change does not seem relevant to the change at hand?

Reply:
Nope, old features again. I basically have two choices here: Revert the changes I don't think is related to my changes, and hope I don't mess up, and leave the problem to the next guy that needs to implement something. Or accept that there's unrelated changes but at least come out with features that's completely up to date.

I prefer the latter. Scouts rule: leave things better than you found them.

Reply:
OK - but when creating the PR for the final review we have to be explicit about the fact that such changes are included and why we choose to do so.